### PR TITLE
Adding overlay layers and an additional base map layer. Separating the CalMAPPER changes into a separate pull request.

### PR DESCRIPTION
**Pull request by elsiel23**
Screen Shot 2022-10-31 at 1 44 15 PM

### CODE REVIEW

**Review by riecke — APPROVED** [review]

**Comment by thequeenofspades** [comment]
If I understand correctly, these changes won't show up in the frontend because they're changing a map page that only exists in the backend (src/planscape/explore/templates/explore.html). I verified this by checking out the branch and running the Angular project. Before we merge this in, maybe we can sync and go over how to get these changes showing up in the Angular map view?

File: src/planscape/explore/static/map.js [hunk 1]
```diff
@@ -3,50 +3,95 @@ const hillshade = L.tileLayer('https://api.mapbox.com/styles/v1/tsuga11/ckcng1sj
...
+controlLayers.setPosition('bottomleft')
+render_huc12();
+render_calmapper_layer();
+
+
+// Render functions
+async function render_calmapper_layer() {
+  var data = JSON.parse(existing_projects)
+
+  // I believe this step makes the map less responsive
```
Comment (jeshlee121) [inline src/planscape/explore/static/map.js:39]: Can you specify who the "I" is for future reference :)

**Review by jeshlee121 — APPROVED** [review]
PSA - I'm still learning how to use Django, so not the best reviewer at the moment; might be worth getting another glance from someone else if you feel that's necessary!

File: src/planscape/explore/static/map.js (hunk 1 above)
Comment (elsiel23) [reply src/planscape/explore/static/map.js]: Moved to another file and added an [elsieling] tag

**Review by thequeenofspades — CHANGES_REQUESTED** [review]
FYI this PR is changing 5,000+ files at the moment because the node_modules folder accidentally got checked in. I'm not sure how that happened, since it should be part of .gitignore, but you should remove it (git rm <path_to_file> might work). I can only see the first 3,000 changed files in the web UI, so I'm not able to see your actual code changes.

**Comment by elsiel23** [comment]
Sorry about that, removed those files from the commit

**Comment by thequeenofspades** [comment]
Sorry, still trying to understand--are the changes in src/planscape/explore related to the changes in src/interface at all? If you revert everything in src/planscape/explore, will the overlay layers and additional base map layer still appear in the Angular application?

**Comment by elsiel23** [comment]
The changes in interface/ and explore/ are unrelated. I only removed my code in explore/ to dedup it. Jon mentioned that he was going to delete the entire explore/ application soon.

**Review by thequeenofspades — APPROVED** [review]

**Pull request merged**